### PR TITLE
Item loan history

### LINKED ISSUE #73: Back-office UI: show loans history box on item details page
Create loans history box on item details page as in the [mockup](https://invenio.mybalsamiq.com/projects/cdsbooks/3%20-%20Item%20Page)
connected with #27

### PR DESCRIPTION
**Pull request by kpsherva**
(closes #73 )
to be merged after #108

### CODE REVIEW

**Review by ntarocco — DISMISSED** [review]

**Review by zzacharo — APPROVED** [review]

File: invenio_app_ils/ui/src/invenio_app_ils/common/urls.js [hunk 1]
```diff
@@ -13,13 +13,18 @@ export const BackOfficeURLS = {
   patronDetails: `${BackOfficeBase}/users/:userPid`,
 };
 
-export const viewDetailsClickUrl = loanPid => {
+export const viewLoanDetailsClickUrl = loanPid => {
```
Comment (zzacharo) [inline invenio_app_ils/ui/src/invenio_app_ils/common/urls.js]: Can you just omit the `Click` in the name?

File: invenio_app_ils/ui/src/invenio_app_ils/common/urls.js [hunk 2]
```diff
@@ -13,13 +13,18 @@ export const BackOfficeURLS = {
...
   return generatePath(BackOfficeURLS.loanDetails, {
     loanPid: loanPid,
   });
 };
 
-export const showAllClickUrl = (documentPid, itemPid, state, patronPid) => {
+export const showAllLoansClickUrl = (
```
Comment (zzacharo) [inline invenio_app_ils/ui/src/invenio_app_ils/common/urls.js]: same here?

**Review by zzacharo — DISMISSED** [review]

**Comment by zzacharo** [comment]
@kprzerwa Can you rebase and just take into account my small naming comment? We are ready to merge it after!

File: invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/ItemDetails/components/ItemPastLoans/state/actions.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    });
+
+    await loanApi
+      .fetchLoans(
+        documentPid,
+        itemPid,
+        null,
+        null,
+        null,
+        'ITEM_RETURNED OR state:CANCELLED'
```
Comment (ntarocco) [inline invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/ItemDetails/components/ItemPastLoans/state/actions.js]: Shouldn't it be also `state:ITEM_RETURNED`?

File: invenio_app_ils/ui/src/invenio_app_ils/common/urls.js (hunk 1 above)
Comment (ntarocco) [reply invenio_app_ils/ui/src/invenio_app_ils/common/urls.js]: Yeah, I guess this function is more about creating/generating an URL Path given some params. It should be named accordingly.

File: invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/ItemDetails/components/ItemPastLoans/state/actions.js (hunk 3 above)
Comment (kpsherva) [reply invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/ItemDetails/components/ItemPastLoans/state/actions.js]: nope, this is already in the function which creates the query. I just avoided here passing the array of few states.

**Review by ntarocco — APPROVED** [review]

**Review by zzacharo — APPROVED** [review]

**Pull request merged**